mindsdb/integrations/handlers/openai_handler/openai_handler.py

```diff
@@ -16,10 +16,10 @@
 
 from mindsdb.utilities.hooks import before_openai_query, after_openai_query
 from mindsdb.utilities import log
-from mindsdb.utilities.config import Config
 from mindsdb.integrations.libs.base import BaseMLEngine
 from mindsdb.integrations.handlers.openai_handler.helpers import retry_with_exponential_backoff, \
     truncate_msgs_for_token_limit
+from mindsdb.integrations.utilities.handler_utils import get_api_key
 
 CHAT_MODELS = ('gpt-3.5-turbo', 'gpt-3.5-turbo-0301', 'gpt-4', 'gpt-4-0314', 'gpt-4-32k', 'gpt-4-32k-0314')
```

Comment:
Not necessarily in the scope of this PR, but would be great to update the `-0314` and `-0301` models with the most recent counterparts listed [here](https://platform.openai.com/docs/models/moderation).

Reply:
Agreed, opened up #6925 for tracking